modules/puzzle/src/main/Attempt.scala

```diff
@@ -34,40 +30,34 @@ object Attempt {
     val date = "d"
     val win = "w"
     val time = "t"
```

Comment:
It does in the real world where if you spend too much time on a move you time out. We may not base tactics ratings on time, but when playing real games time is very important and being able to identify where you have time trouble is very important for deciding what areas to focus on studying (esp. When we allow custom puzzle sets)